### PR TITLE
No Issue : refactored c_annotations.py file constructor 

### PR DESCRIPTION
**Pull request by DeepakSingh260**

### CODE REVIEW

File: Doc/tools/extensions/c_annotations.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def __init__(self, refcount_filename, stable_abi_file):
         self.stable_abi_data = {}
         with open(stable_abi_file, 'r') as fp:
             for record in csv.DictReader(fp):
-                role = record['role']
+                #role = record['role']
```
Comment (arhadthedev) [inline Doc/tools/extensions/c_annotations.py:84]: What is the purpose of leaving `role` uninitialized? I know nothing about the documentation generator so clarification would be great.

**Comment by sobolevn** [comment]
Hi, @DeepakSingh260 👋 

If you want to contribute, please:
1. Open an issue first, describe what is the point of doing what you are doing. And how it will help the project
2. Wait untill some from the core-dev team will approve your plan
3. Send a PR 

Untill then, I am going to close it.

And you can always work on some existing issue, we have 5k of them!

File: Doc/tools/extensions/c_annotations.py (hunk 1 above)
Comment (DeepakSingh260) [reply Doc/tools/extensions/c_annotations.py:84]: The role variable was initialized but was never used .Probably at one time it was being used but not anymore.

Comment (arhadthedev) [reply Doc/tools/extensions/c_annotations.py:84]: Thank you, I see now. However, small edits not fixing bugs just clutter the changelog so they are not worth bothering.

**Pull request closed without merging**